apps/demos/Demos/Chat/Customization/test-code.js

```diff
@@ -0,0 +1,3 @@
+testUtils
```

Comment:
You don't need this file because, in your demo, the height of the Chat component fits within the test page dimensions (1000 × 800).